lib/formz.dart

```diff
@@ -96,10 +102,6 @@ abstract class FormzInput<T, E> {
   /// [value] is invalid and `null` otherwise.
   E? validator(T value);
 
-  /// The error to display if the [FormzInput] value
-  /// is not valid and has been modified.
-  E? get displayError => isPure ? null : error;
```

Comment:
@jorgecoca @definitelyokay I ended up removing `displayError` in light of our prior discussions given it wasn't really useful in the context of the example and it's not hard to write a custom extension if it's needed for a particular use-case. See https://github.com/VeryGoodOpenSource/formz/pull/52/files?show-deleted-files=true&show-viewed-files=true&file-filters%5B%5D=#diff-72fd825ad17ce7800a2078e06f0dff1e12836ee59760465e78d9a5d222cebd15R229 for the example usage.